test/test_torch.py

```diff
@@ -7535,6 +7535,18 @@ def test_flip(self, device):
         self.assertEqual(torch.tensor([7, 8, 5, 6, 3, 4, 1, 2]).view(2, 2, 2), data.flip(0, 1))
         self.assertEqual(torch.tensor([8, 7, 6, 5, 4, 3, 2, 1]).view(2, 2, 2), data.flip(0, 1, 2))
 
+        # Complex Dtypes
+        def test_complex_flip(dtype):
+            complex_data = torch.tensor([1, 2, 3, 4, 5, 6, 7, 8], device=device, dtype=dtype).view(2, 2, 2)
```

Comment:
These complex values aren't very interesting since they all have zero imaginary part. Instead, I suggest you generate random tensors of various sizes and compare the actual torch result of flipping to the expected NumPy result.

Reply:
agreed